apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.scss

```diff
@@ -34,5 +33,11 @@ input.form-control.date-range-picker {
     border: 1px solid $chetwode-blue;
     border-radius: 4px;
     padding: 2px;
-    top: 0 !important;
+    top: 0px !important;
+}
+.giddh-datepicker.date-range-picker {
+    padding-left: 45px !important;
+}
+.modal-header > .close {
```

Comment:
please add scss, not css @ravinderwalkover

Reply:
Done